vm/Cargo.toml

```diff
@@ -59,6 +59,7 @@ unic = "0.9.0"
 maplit = "1.0"
 proc-macro-hack = { version = "0.5", optional = true }
 bitflags = "1.1"
+libc = "0.2"
```

Comment:
So this was inevitable:). We all want rust instead of c, but, oh well :P

Reply:
I could use just raw integer literals instead of the libc constants, they're probably not likely to change often. It's just tough when the `errno` module from CPython looks like [this](https://github.com/python/cpython/blob/master/Modules/errnomodule.c).